### PR TITLE
feat: bump to sdk-v2 and take advantage of new relayer fee curve

### PR DESCRIPTION
**Pull request by nicholaspai**
## Needs QA

**Overview**

Input amounts to send (on any network, network doesn't matter) for different tokens and see if fees charged make sense. Previously the dapp is charging 0.03% on all relays. The new curve defined for tokens in `src/utils/constants` will charge an increasing fee for the tokens with entries. If there is no entry for a token, then the flat 0.03% should be charged.

**Acceptance Criteria**

TODO

### How To Report QA
Testers, use the following templates to pass the pull request or to report issues that require a resolution.
Following these guidelines will help the developer respond more quickly to issues, but you are free to comment
in any way as needed to help resolve issues.

**Reporting QA Issues**

Testers should use Github comments to report issues with the PR. Use screenshots if relevant, you can also
check the [developer console](https://support.airtable.com/hc/en-us/articles/232313848-How-to-open-the-developer-console)
to see if there are any logs which might be related. Use the following template when reporting a problem or issue with the PR.
You submit multiple comments using this template if there are multiple issues.

```
**QA Issue**

<!--
Describe the problem with you ran into, if it has to do with one of the acceptance criteria, then add it here.
Add any screenshots of the app or console logs if you think it would be useful.
-->

**Repro Steps**

<!--
If this issue was seen after taking particular steps, then note how to do it here.

Example:
1. Go to bridge tab and send a tx between any chains.
2. go to pool tab before going to transaction tab.
3. see that my pending transaction is missing.
-->

**Testing Environment**

<!--
Optionally include anything about your testing environment, ie phone, tablet, chrome or safari, if you used
metamask or a different wallet, etc. This will let other testers know if there are configurations they could
test differently from yours.

Example:
Used Metamask and the latest version of Safari on a Macbook. 
-->


```

**Reporting QA Regression**

A regression is something that was working, but now is no longer working due to the new code changes. The breakage
may seem completely unrelated to the pull request, which is why they can go unnoticed during testing.
If you find a regression, or arent sure, report it anyway as an issue and the engineer can decide how to resolve it. 

**Reporting QA Passed**

If you tested and did not find any issues, you will need to signal to the developer that this passed your QA.
Do this by adding comments in the pr. You should communicate to both the developers and other testers what you did during testing. Use the following
template to signal that QA passed your testing.

```
**QA Pass**

<!--
Describe what you tested, if you tested beyond what was originally specified, or any other parts of the app.
This will help others decide how to do their testing, and let the engineer know the testing coverage of the app.

Example:
Tested sending eth and weth between optimism and arbitrum on kovan and back again. Both cases I was able to see my 
transaction and speed it up. No issues with the UI but I wasnt sure if the transaction actually was faster. Also
tested adding and removing liquidity from the pool tab which worked as expected. 

-->

**Testing Environment**

<!--
Optionally include anything unique about your testing environment, ie phone, tablet, chrome or safari, if you used
Metamask or a different wallet, etc. This will let other testers know if there are configurations they could
test differently from yours.

Example:
Used Metamask and the latest version of Safari on a Macbook. 
-->

```

### CODE REVIEW

**Comment by nicholaspai** [comment]
@daywiss @Tulun any idea why this is failing to build?

File: api/constants.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { ethers } from "ethers";
+
+export const relayerFeeCapitalCostConfig = {
```
Comment (daywiss) [inline api/constants.js:3]: is this duplicate of whats in the utils file? if so could we try to dedupe

**Review by daywiss — COMMENTED** [review]
taking a look at why the build fails

File: api/constants.js (hunk 1 above)
Comment (nicholaspai) [reply api/constants.js:3]: It is but I thought we can't import from outside `src`into `api`,let me try again

**Review by Tulun — APPROVED** [review]
See s fine. Build is also no longer failing?
